core/container/bit_array/bit_array.odin

```diff
@@ -27,25 +27,24 @@ Bit_Array_Iterator :: struct {
 }
 
 /*
-	In:
-		- ba:   ^Bit_Array - the array to iterate over
+In: <br>
```

Comment:
Please don't use `<br>` within the documentation. The docs need to be readable from the test and from `odin doc` which is a terminal viewer.

We should use a better markup to make it clear that it is a list below